src/GafferScene/EditScopeAlgo.cpp

```diff
@@ -854,3 +855,210 @@ const Gaffer::GraphComponent *GafferScene::EditScopeAlgo::attributeEditReadOnlyR
 
 	return nullptr;
 }
+
+// Set Membership
+// ==============
+
+namespace
+{
+
+static int g_addSetColumnIndex = 0;
+static int g_removeSetColumnIndex = 1;
+
+SceneProcessorPtr setMembershipProcessor()
+{
+	SceneProcessorPtr result = new SceneProcessor( "SetMembershipEdits" );
+
+	SpreadsheetPtr spreadsheet = new Spreadsheet;
+	result->addChild( spreadsheet );
+	spreadsheet->selectorPlug()->setValue( "${setMembership:set}" );
+	spreadsheet->rowsPlug()->addColumn( new StringVectorDataPlug( "add", Plug::Direction::In, new StringVectorData() ) );
```

Comment:
Would there be any merit in having the column names matching with the SetMembership enum values?

Reply:
Fixed in 5b8d879faa5a588fd142817501fab1638043f670